src/components/header/header_alert/header_alert.tsx

```diff
@@ -22,36 +26,27 @@ export const EuiHeaderAlert: FunctionComponent<EuiHeaderAlertProps> = ({
   date,
   text,
   title,
+  badge,
   ...rest
 }) => {
   const classes = classNames('euiHeaderAlert', className);
 
   return (
-    <EuiI18n token="euiHeaderAlert.dismiss" default="Dismiss">
-      {(dismiss: string) => (
-        <div className={classes} {...rest}>
-          <EuiButtonIcon
-            aria-label={dismiss}
-            iconType="cross"
-            size="s"
-            className="euiHeaderAlert__dismiss"
-          />
-
-          <div className="euiHeaderAlert__title">{title}</div>
-
-          <div className="euiHeaderAlert__text">{text}</div>
-
-          <EuiFlexGroup justifyContent="spaceBetween">
-            <EuiFlexItem grow={false}>
-              <div className="euiHeaderAlert__action euiLink">{action}</div>
-            </EuiFlexItem>
-
-            <EuiFlexItem grow={false}>
-              <div className="euiHeaderAlert__date">{date}</div>
-            </EuiFlexItem>
-          </EuiFlexGroup>
-        </div>
+    <div className={classes} {...rest}>
+      <EuiFlexGroup justifyContent="spaceBetween">
+        <EuiFlexItem>
+          <div className="euiHeaderAlert__date">{date}</div>
+        </EuiFlexItem>
+        {badge ? <EuiFlexItem grow={false}>{badge}</EuiFlexItem> : undefined}
+      </EuiFlexGroup>
+
+      <div className="euiHeaderAlert__title">{title}</div>
```

Comment:
Can we force developers to pass in a heading level for this?

```suggestion
      <TitleHeading className="euiHeaderAlert__title">{title}</TitleHeading>
```

Outside of the `return` statement you'd need to accept a `titleHeading` prop (or some other name).

Reply:
While I agree with this, let's save it for a future change. As it stands, this component will only be used in one place within Kibana and it accepts a JSON feed where the author simply defines a `title` value. Example here: https://github.com/elastic/newsfeeds/blob/master/data/ess/ess-training.yaml

In the meantime, @andreadelrio and I chatted and she'll replace the current div with a heading element. We can revisit the need to handle user-defined heading levels at a future date when/if the need arises.

Reply:
I've added `h3` for titles since the flyout title is using `h2`.